terraform/modules/network/resources.tf

```diff
@@ -26,56 +26,63 @@ resource "aws_security_group" "default" {
       from_port   = 0
       to_port     = 0
       protocol    = "-1"
-      cidr_blocks = concat(split(",", var.config.ip_whitelist), ["10.0.0.0/16"])
+      cidr_blocks = ["10.0.0.0/16"]
+    }
+
+   ingress {
+      from_port   = -1
+      to_port     = -1
+      protocol    = "icmp"
+      cidr_blocks = split(",", var.config.ip_whitelist)
     }
 
    ingress {
       from_port   = 22
       to_port     = 22
       protocol    = "tcp"
-      cidr_blocks = concat(split(",", var.config.ip_whitelist), ["10.0.0.0/16"])
+      cidr_blocks = split(",", var.config.ip_whitelist)
```

Comment:
This is how I had it in my original PR. For some reason @P4T12ICK added this limitation to addresses 10.0.0.0/16

Reply:
for ssh 10.0.0.0 is not needed but for the different splunk ports it is needed